pkg/notes/notes_data.go

```diff
@@ -513,3 +532,74 @@ func setEmptyComponentGitHubRepositoryFromOtherKilnfile(k1, k2 cargo.Kilnfile) c
 }
 
 func closeAndIgnoreError(c io.Closer) { _ = c.Close() }
+
+type TrainstatFetcher interface {
+	FetchTrainstatNotes(milestone string, version string, tile string) ([]string, error)
+}
+
+type TrainstatClient struct {
+	host       string
+	httpClient *http.Client
+}
+
+func NewTrainstatClient(host string) TrainstatClient {
```

Comment:
is quoted "null" intentional here?

Reply:
Oh, I didn't realize about this earlier, thanks for pointing this out, that condition is not necessary, I'll delete it.